insights/specs/default.py

```diff
@@ -223,6 +224,24 @@ def is_ceph_monitor(broker):
     cinder_api_log = first_file(["/var/log/containers/cinder/cinder-api.log", "/var/log/cinder/cinder-api.log"])
     cinder_conf = first_file(["/var/lib/config-data/puppet-generated/cinder/etc/cinder/cinder.conf", "/etc/cinder/cinder.conf"])
     cinder_volume_log = first_file(["/var/log/containers/cinder/volume.log", "/var/log/containers/cinder/cinder-volume.log", "/var/log/cinder/volume.log"])
+
+    @datasource(HostContext)
+    def cloud_init_network(broker):
```

Comment:
I think `cloud_cfg` would be a better spec name since that is the source file that is being collected.  Filtering is being performed here, but its still the `cloud.cfg` file.

Reply:
@bfahr  Sure.